web/views/view_helpers.ex

```diff
@@ -0,0 +1,64 @@
+defmodule App.ViewHelpers do
+  @moduledoc """
+    Functions to transform html as stored in the database by wagtail into
+    the html we actually render on the site
+  """
+
+  import App.Router.Helpers
+  import Ecto.Query, only: [from: 2]
+
+  alias App.CMSRepo
+
+  # Regex to match wagtail embedded image and pull out title, id and format
+  @embed_regex ~r/<embed\salt=\"([^\"]+)\"\sembedtype=\"image\"\sformat=\"([^\"]+)\"\sid=+\"([^\"]+)\"\/>/
+  @link_regex ~r/<a\sid=\"([^\"]+)\"\slinktype=\"page\">([^<]+)<\/a>/
+
+  def renderImage(html_string, conn) do
```

Comment:
It seems a convention to have the conn map as the first function argument.

Maybe switching this may help with piping as I doubt you'll be piping in the regex, but you could be piping in the `conn`